app/src/main/java/com/kinandcarta/create/proxytoggle/injection/AppModule.kt

```diff
@@ -1,25 +1,26 @@
 package com.kinandcarta.create.proxytoggle.injection
 
+import com.kinandcarta.create.proxytoggle.android.DeviceSettingsManagerImpl
 import com.kinandcarta.create.proxytoggle.android.SharedPrefsAppSettings
-import com.kinandcarta.create.proxytoggle.broadcast.ProxyUpdateListenerProvider
-import com.kinandcarta.create.proxytoggle.broadcast.ProxyUpdateListenerProviderImpl
-import com.kinandcarta.create.proxytoggle.settings.AppSettings
+import com.kinandcarta.create.proxytoggle.core.android.DeviceSettingsManager
+import com.kinandcarta.create.proxytoggle.core.settings.AppSettings
 import dagger.Binds
 import dagger.Module
 import dagger.hilt.InstallIn
-import dagger.hilt.android.components.ApplicationComponent
+import dagger.hilt.components.SingletonComponent
 
+@Suppress("unused")
```

Comment:
If this is detekt a nicer way would be to tweak the yml file to exclude warnings for this?

Reply:
Is the suppression useful then?

Reply:
well yeah, cause it doesn't show this as an issue in the general issues. Ideally, we'd have 0 issues in there.